PowerShellGet/PSModule.psm1

```diff
@@ -6527,6 +6559,14 @@ function Validate-VersionParameters
                     -CallerPSCmdlet $CallerPSCmdlet `
                     -ErrorCategory InvalidArgument
     }
+    elseif( (($MinimumVersion -match '-') -or ($MaximumVersion -match '-') -or ($RequiredVersion -match '-')) -and -not $AllowPrerelease)

```

Comment:
>elseif( (($MinimumVersion -match '-') -or ($MaximumVersion -match '-') -or ($RequiredVersion -match '-')) -and -not $AllowPrerelease) [](start = 4, length = 133)

As these parameters type got changed to string, it is required to validate these version parameters in this function.